nmdc_runtime/site/ops.py

```diff
@@ -591,9 +639,41 @@ def id_minter(*args, **kwargs):
     return database
 
 
+@op(
+    required_resource_keys={"nmdc_portal_api_client"},
+    config_schema={"submission_id": str},
+)
+def fetch_nmdc_portal_submission_by_id(context: OpExecutionContext) -> Dict[str, Any]:
+    submission_id = context.op_config["submission_id"]
+    client: NmdcPortalApiClient = context.resources.nmdc_portal_api_client
+    return client.fetch_metadata_submission(submission_id)
+
+
+@op(required_resource_keys={"runtime_api_site_client"})
+def translate_portal_submission_to_nmdc_schema_database(
+    context: OpExecutionContext,
+    metadata_submission: Dict[str, Any],
+) -> nmdc.Database:
+    client: RuntimeApiSiteClient = context.resources.runtime_api_site_client
+
+    def id_minter(*args, **kwargs):
+        response = client.mint_id(*args, **kwargs)
+        response.raise_for_status()
```

Comment:
`response.raise_for_status()` is redundant now, as I added it to `RuntimeApiUserClient.request`. 🙂 Please remove here.